sdk/apps/modal-example/src/routes/aleph.tsx

```diff
@@ -0,0 +1,120 @@
+import { getPolkadotWallets, NightlyConnectAdapter } from '@nightlylabs/wallet-selector-polkadot'
+import { ApiPromise, WsProvider } from '@polkadot/api'
+import { createEffect, createSignal, onMount, Show } from 'solid-js'
+import { Title } from 'solid-start'
+import toast from 'solid-toast'
+
+const RECEIVER = '5CFRopxy991HCJj1HYtUQjaaBMw9iRLE9jxPndBsgdCjeJj5'
+
+export default function Polkadot() {
+  const [adapter, setAdapter] = createSignal<NightlyConnectAdapter>()
+  const [eager, setEager] = createSignal(false)
+  const [publicKey, setPublicKey] = createSignal<string>()
+  const [api, setApi] = createSignal<ApiPromise>()
+  const provider = new WsProvider('wss://ws.test.azero.dev/')
+
+  onMount(async () => {
+    const adapter = NightlyConnectAdapter.buildLazy(
+      {
+        appMetadata: {
+          name: 'NCTestSui',
+          description: 'Nightly Connect Test',
+          icon: 'https://docs.nightly.app/img/logo.png',
+          additionalInfo: 'Courtesy of Nightly Connect team'
+        }
+      },
+      true,
+      document.getElementById('modalAnchor')
+    )
+
+    adapter.canEagerConnect().then((canEagerConnect) => {
+      setEager(canEagerConnect)
+    })
+    setAdapter(adapter)
+
+    ApiPromise.create({
+      provider
+    }).then((api) => {
+      setApi(api)
+    })
+  })
+  createEffect(() => {
+    if (eager()) {
+      adapter()
+        ?.connect()
+        .then(
+          async () => {
+            const accounts = await adapter()!.accounts.get()
+            console.log(accounts)
+            setPublicKey(accounts[0].address)
+            console.log('connect resolved successfully')
+          },
+          () => {
+            console.log('connect rejected')
+          }
+        )
+    }
+  })
+
+  return (
+    <main>
+      <Title>Sui Example</Title>
+      <div id="modalAnchor" />
+      <Show
+        when={!!publicKey()}
+        fallback={
+          <button
+            onClick={async () => {
+              console.log(getPolkadotWallets())
+              await adapter()!.connect()
+              const accounts = await adapter()!.accounts.get()
+              console.log(accounts)
+              setPublicKey(accounts[0].address)
+              console.log('adapter', adapter())
+            }}>
+            Connect
+          </button>
+        }>
+        <h1>Current address: {publicKey()}</h1>
+        <button
+          onClick={async () => {
+            try {
+              const payload = api()!.tx.balances.transfer(RECEIVER, 500)
+              const signed = await payload.signAsync(publicKey()!, { signer: adapter()!.signer })
+              console.log({ signed })
+              toast.success('Transaction was signed and sent!')
```

Comment:
there is no sending transaction here; it would be good to have it to verify if everything is ok with signed transaction